src/template/component.ts

```diff
@@ -24,7 +30,41 @@ ${name}.propTypes = {
 export default ${name};
 `;
 
-export const component = name => `import { React, PropTypes } from '../../dependencies';
+export const felaHookComponent = (name: string, dependencies: boolean) => `${
+  dependencies
+    ? "import { React, PropTypes } from '../../dependencies';"
+    : "import React from 'react';\nimport PropTypes from 'prop-types';"
+}
+
```

Comment:
Maybe add draft of hook import `import useFelaEnhanced from ''`

Reply:
Just noticed it's already present in typescript templates so pls add it here too 🙏  `import { useFelaEnhanced } from 'hooks'`

Reply:
I forgot to add it there 👍